File: src/sentry.py
import sentry_sdk

SENTRY_DSN = "https://f0e4038c64dc4c79b5404c93608df210@o184107.ingest.sentry.io/5780285"


def setup_sentry():
    sentry_sdk.init(SENTRY_DSN, traces_sample_rate=1.0)

File: mosaic-streamlit/src/sentry.js
const { init } =
  process.type === "browser"
    ? require("@sentry/electron/dist/main")
    : require("@sentry/electron/dist/renderer");

const SENTRY_DSN = "https://f0e4038c64dc4c79b5404c93608df210@o184107.ingest.sentry.io/5780285"

const setupSentry = () => {
  init({ dsn: SENTRY_DSN })
}

export default setupSentry
